diem-move/diem-framework/DPN/releases/artifacts/current/build/DPNFramework/sources/AutoPay.move

```diff
@@ -51,8 +50,8 @@ address DiemFramework {
     const INVALID_PERCENTAGE: u64 = 010022;
     /// Attempt to use a UID that is already taken 
     const UID_TAKEN: u64 = 010023;
-    /// Attempt to make a payment to a non-community-wallet 
-    const PAYEE_NOT_COMMUNITY_WALLET: u64 = 010024;
+    // /// Attempt to make a payment to a non-community-wallet 
```

Comment:
Could you add why this is commented out and what to do with it in the future? Delete, keep, update?

Reply:
Oh yes sorry, did not see they were in releases folder.